src/app/assets/containers/asset-detail/asset-detail.component.ts

```diff
@@ -91,6 +96,10 @@ export class AssetDetailComponent implements OnInit {
       this.addresses$ = this.store.pipe(select(fromStore.getAllAddressesForAssetId(theId)));
     });
 
+    // Initialize each property state in view mode
+    this.defendants$.subscribe(defendants => defendants
```

Comment:
This has a memory leak since it'll listen to it forever - there is no unsubscribe for it. Can you add a `.pipe(take(1))` before `.subscribe()`?